src/ontology/fbbt-edit.obo

```diff
@@ -121422,6 +121426,425 @@ relationship: part_of FBbt:00100311 ! cortical zone of lymph gland primary lobe
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00058165 ! metathoracic leg disc epithelium proper
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058178
+name: epithelial cell of genital disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00058166 ! genital disc epithelium proper
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058179
+name: epithelial cell of male genital disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00058168 ! male genital disc epithelium proper
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058180
+name: epithelial cell of female genital disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00058167 ! female genital disc epithelium proper
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058181
+name: peripodial epithelium of prothoracic leg disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00001762 ! peripodial epithelium
+intersection_of: part_of FBbt:00001781 ! prothoracic leg disc
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058182
+name: peripodial epithelium of mesothoracic leg disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00001762 ! peripodial epithelium
+intersection_of: part_of FBbt:00001782 ! mesothoracic leg disc
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058183
+name: peripodial epithelium of metathoracic leg disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00001762 ! peripodial epithelium
+intersection_of: part_of FBbt:00001783 ! metathoracic leg disc
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058184
+name: peripodial epithelium of male genital disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00001762 ! peripodial epithelium
+intersection_of: part_of FBbt:00001785 ! male genital disc
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058185
+name: peripodial epithelium of female genital disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00001762 ! peripodial epithelium
+intersection_of: part_of FBbt:00001787 ! female genital disc
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-06-03T18:57:00Z
+
+[Term]
+id: FBbt:00058186
+name: epithelial cell of peripodial epithelium of labial disc
```

Comment:
these 'epithelial cell of peripodial epithelium of X disc' names seem unnecessarily long - I think 'peripodial epithelial cell of X disc' would be sufficient